File: TP3_Python/Codingan/MONITOR.py
from External import External

class MONITOR(External):
    def __init__(self, resolusi="", ukuran=0, koneksi="", merk_external=""):
        super().__init__(koneksi, merk_external)
        self.resolusi = resolusi
        self.ukuran = ukuran

    def set_resolusi(self, resolusi):
        self.resolusi = resolusi

    def set_ukuran(self, ukuran):
        self.ukuran = ukuran

    def get_resolusi(self):
        return self.resolusi

    def get_ukuran(self):
        return self.ukuran

File: TP3_CPP/Codingan/MONITOR.cpp
#pragma once
#include "External.cpp"
#include <iostream>
#include <string>

using namespace std;

class MONITOR : public External
{
private:
    string Resolusi;
    int Ukuran;

public:
    MONITOR()
    {
    }

    MONITOR(string Resolusi, int Ukuran, string Koneksi, string MerkExternal) : External(Koneksi, MerkExternal)
    {
        this->Resolusi = Resolusi;
        this->Ukuran = Ukuran;
    }

    // Setter
    void setResolusi(string Resolusi)
    {
        this->Resolusi = Resolusi;
    }

    void setUkuran(int Ukuran)
    {
        this->Ukuran = Ukuran;
    }

    // Getter
    string getResolusi()
    {
        return Resolusi;
    }

    int getUkuran()
    {
        return Ukuran;
    }

    ~MONITOR()
    {
    }
};
